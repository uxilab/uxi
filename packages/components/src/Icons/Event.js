import React from 'react';
import PropTypes from 'prop-types';
import SvgIcon from '../SvgIcon';
/* eslint-disable max-len */

const Event = props => (
  <SvgIcon {...props}>
    <svg
      focusable="false"
      xmlns="http://www.w3.org/2000/svg"
      viewBox="0 0 14.57 16"
      width="24px"
      height="24px"
    >
      <g id="Layer_2" data-name="Layer 2">
        <g id="Layer_1-2" data-name="Layer 1">
          <path d="M5.34.73a.73.73,0,1,0-1.46,0v2.6a.73.73,0,1,0,1.46,0Z" />
          <path d="M10.7.73a.73.73,0,1,0-1.46,0v2.6a.73.73,0,1,0,1.46,0Z" />
          <rect x="6.44" y="1.4" width="1.7" height="1.25" />
          <path d="M14.57,2.89a1.46,1.46,0,0,0-1.46-1.46H11.68V2.68h1.65V6.13H1.25V2.68H2.93V1.43H1.46A1.46,1.46,0,0,0,0,2.89V14.54A1.46,1.46,0,0,0,1.46,16H13.11a1.46,1.46,0,0,0,1.46-1.46ZM1.25,14.75V7.59H13.32v7.16Z" />
        </g>
      </g>
    </svg>
  </SvgIcon>
);

Event.propTypes = {
  color: PropTypes.string,
};

Event.defaultProps = {
  color: '#6d6d71',
};

export default Event;
