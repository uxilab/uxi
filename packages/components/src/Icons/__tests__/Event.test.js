import React from 'react';
import Event from '../Event';

describe('Icon : <Event />', () => {
  it('match snapshot', () => {
    expect(shallow(<Event />)).toMatchSnapshot();
  });
});

