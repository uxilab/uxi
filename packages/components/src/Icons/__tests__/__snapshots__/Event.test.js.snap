// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon : <Event /> match snapshot 1`] = `
<SvgIcon
  color="#6d6d71"
  hoverColor="#6d6d71"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  size={24}
  style={Object {}}
  viewBox="0 0 24 24"
>
  <svg
    focusable="false"
    height="24px"
    viewBox="0 0 14.57 16"
    width="24px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      data-name="Layer 2"
      id="Layer_2"
    >
      <g
        data-name="Layer 1"
        id="Layer_1-2"
      >
        <path
          d="M5.34.73a.73.73,0,1,0-1.46,0v2.6a.73.73,0,1,0,1.46,0Z"
        />
        <path
          d="M10.7.73a.73.73,0,1,0-1.46,0v2.6a.73.73,0,1,0,1.46,0Z"
        />
        <rect
          height="1.25"
          width="1.7"
          x="6.44"
          y="1.4"
        />
        <path
          d="M14.57,2.89a1.46,1.46,0,0,0-1.46-1.46H11.68V2.68h1.65V6.13H1.25V2.68H2.93V1.43H1.46A1.46,1.46,0,0,0,0,2.89V14.54A1.46,1.46,0,0,0,1.46,16H13.11a1.46,1.46,0,0,0,1.46-1.46ZM1.25,14.75V7.59H13.32v7.16Z"
        />
      </g>
    </g>
  </svg>
</SvgIcon>
`;
